qiskit_experiments/library/characterization/t2ramsey.py

```diff
@@ -127,14 +148,23 @@ def circuits(self, backend: Optional[Backend] = None) -> List[QuantumCircuit]:
         elif self.experiment_options.unit != "s":
             conversion_factor = apply_prefix(1, self.experiment_options.unit)
 
+        self.set_analysis_options(
+            extra={
+                "osc_freq": self.experiment_options.osc_freq,
+                "conversion_factor": conversion_factor,
```

Comment:
Same for T1. Probably we need to keep `osc_freq`, but I don't think other info is really necessary. We can even remove `osc_freq` here by subtracting `osc_freq` from fit frequency value before adding it to the data base entry.

Reply:
Regarding the conversion factor, if you remove it, how do you intend to know which unit is used?